Vagrantfile

```diff
@@ -18,6 +18,9 @@ Vagrant.configure("2") do |config|
   config.vm.define "2016-box", autostart: false do |cfg|
     cfg.vm.box     = "StefanScherer/windows_2016_docker"
     cfg.vm.provision "shell", path: "scripts/create-machine.ps1", args: "-machineHome #{home} -machineName 2016-box"
+    cfg.vm.provider "virtualbox" do |v, override|
+      override.vm.network :private_network, ip: "192.168.59.50", gateway: "192.168.56.1"
```

Comment:
We copied those lines from the original place down below. However, regardless of the fact that it works perfectly, we have been wondering if the settings are correct.

Is it correct that the host address is `192.168.59.x`, but the gateway address is `192.168.56.1`? Disclaimer: We don't know much about VirtualBox internals, so it might be perfectly fine. Just asking ;].

Reply:
Hi again,

we just discovered #79 and #80, submitted by @jaraco the other day, where this change was coming from.

```diff
-    override.vm.network :private_network, ip: "192.168.99.90", gateway: "192.168.99.1"
+    override.vm.network :private_network, ip: "192.168.59.90", gateway: "192.168.56.1"
```

Where both addresses have been within the `192.168.99.0` network beforehand, and the intention was to change it to the `192.168.59.0` network, using `192.168.56.1` as the gateway address might have been a typo.

With kind regards,
Andreas.

Reply:
Oh, only now [I am seeing](https://jodies.de/ipcalc?host=192.168.56.0&mask1=21) that `192.168.56.0/21` has a `/21` netmask, so the address range is from `192.168.56.1` to `192.168.63.254`. Apologies for missing that. So, hosts at `192.168.59.x` are completely valid.

> Just asking in order to eventually learn that `192.168.56.1` might be anything special?

So it now makes sense to me that VirtualBox made it so that `192.168.56.1` is the gateway address.

Sorry for the noise about this detail.

Reply:
Thanks for investigating it. I was also wondering, but the netmask is the explanation.